integration_tests/suite/test_auto_create.py

```diff
@@ -39,7 +39,16 @@ def setUp(self):
 
         def wait_for_dird_bus_connection():
             response = self.client.status.get()
-            assert_that(response, has_entries(bus_consumer={'status': 'ok'}))
+            assert_that(
+                response,
+                has_entries(
+                    {
+                        'master_tenant': has_entries(status='ok'),
+                        'rest_api': has_entries(status='ok'),
+                        'bus_consumer': has_entries(status='ok'),
+                    }
+                ),
+            )
 
```

Comment:
Why waiting for everything?

Reply:
I have put it back; thought since we are using `EverythingOkWaitStrategy`; we had to wait for all three to have status `Ok`.